pocs/tests/test_rs232.py

```diff
@@ -32,9 +50,16 @@ def test_detect_bogus_scheme(handler):
         rs232.SerialData(port='bogus://')
 
 
+def test_detect_bogus_scheme(handler):
+    """When our handlers are installed, will still detect unknown scheme."""
```

Comment:
What are valid schemes? Have we documented these handlers and schemes as they apply to us?

Reply:
Rather than document all of those (mostly provided by PySerial), I documented what the scheme must be, and then violated that so that the exception will be raised.